utils/not_doctested.txt

```diff
@@ -626,7 +626,6 @@ src/transformers/models/lilt/configuration_lilt.py
 src/transformers/models/llama/configuration_llama.py
 src/transformers/models/llama/convert_llama_weights_to_hf.py
 src/transformers/models/llama/modeling_llama.py
-src/transformers/models/llama/tokenization_llama_fast.py
```

Comment:
why we need this change ?

Reply:
[These lines in `src/transformers/models/llama/tokenization_llama_fast.py`](https://github.com/huggingface/transformers/pull/26669/files/1231d4188fa5ae8fe0572999fb3903cd5185c347#diff-a7c16a11921618850a92fb91c4c7d4764d1222a1c5c4553b53d1c37a6cb3780aR63-R68) can be tested after this PR merged.

Reply:
oh, you are right. My head is flooded by the many `docstring` PRs ...sorry 😅